website/client/components/guilds/groupFormModal.vue

```diff
@@ -0,0 +1,285 @@
+<template lang="pug">
+  b-modal#modal1(:title="$t('createGuild')", :hide-footer="true")
+    form(@submit.stop.prevent="submit")
+      .form-group
+        label
+          strong {{$t('name')}}*
```

Comment:
where possible I've tried to use v-once for `$t` expression to save some computation time